run-ios-flavor.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# ./run-ios-bash.sh prod - runs the Firebase prod configuration
+# ./run-ios-bash.sh dev	- runs the Firebase dev configuration
```

Comment:
Ah just saw this, please fix the name here

Reply:
changed the name, sorry for that escape